profiles/esha98.json

```diff
@@ -0,0 +1,7 @@
+{
+    "image": "https://user-images.githubusercontent.com/60232135/215237852-c021e76d-316d-4fa0-96dd-59d9666bc6fe.png",
```

Comment:
I recommend more of an image like this. What do you think? 🙂

![image](https://user-images.githubusercontent.com/51391473/215367152-fbb4084f-79a4-470b-a96a-1656f7a735dc.png)

```suggestion
    "image": "https://user-images.githubusercontent.com/51391473/215367152-fbb4084f-79a4-470b-a96a-1656f7a735dc.png",
```

Reply:
Yes, this looks decent. I used your recommended image @Panquesito7